src/geometry/manifold/vector_bundle/basic.lean

```diff
@@ -0,0 +1,271 @@
+/-
+Copyright (c) 2022 Floris van Doorn, Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn, Heather Macbeth
+-/
+import geometry.manifold.cont_mdiff_map
```

Comment:
Only `pullback` uses `cont_mdiff_map`. Could you move `pullback` to a separate file and only import `cont_mdiff` here.
The reason is that we cannot transitively import `manifold/tangent_bundle` (it might *also* be a good idea to split `cont_mdiff_map` so that it doesn't import `tangent_bundle`, but I think reducing this import is also good).